addons/website_mail_channel/views/snippets/s_channel.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_channel" name="Discussion Group">
+    <div class="oe_snippet_body js_follow_alias"
```

Comment:
You can remove oe_snippet_body, this is an automatic class added when using the system correctly ;)